packages/itwin/grouping-mapping-widget/src/widget/components/QueryBuilder.ts

```diff
@@ -166,110 +164,59 @@ export class QueryBuilder {
     propertyValue: Primitives.Value,
     isAspect: boolean,
   ) {
+    const addRelatedPropertyToQuery = this.addRelatedPropertyToQuery(unionIndex, isAspect);
```

Comment:
This looks weird. Both are functions, with the same names. One created a function and another is the created function.

Rename.

Reply:
Changed

Reply:
resolved